reactioncommerce/reaction
# Address review workflow

Closes #2104 

Adds a new step for when address validation returns a different address. Allows them to "merge" the validated address with their entered address.

### How to Test

1. Enable Avalara
1. Add something to cart and checkout
1. Add an address that doesn't validate (which will be pretty much any address)
1. Observe that you can reconcile the address using this new screen
